client/src/components/Page/PageIndexActions.vue

```diff
@@ -0,0 +1,29 @@
+<script setup lang="ts">
+import { BButton } from "bootstrap-vue";
+import { useRouter } from "vue-router/composables";
+
+import { FontAwesomeIcon } from "@fortawesome/vue-fontawesome";
+import { faPlus } from "@fortawesome/free-solid-svg-icons";
+import { library } from "@fortawesome/fontawesome-svg-core";
+
+const router = useRouter();
+library.add(faPlus);
+
+interface PageIndexActionsProps {
+    root: string;
+}
+
+const props = defineProps<PageIndexActionsProps>();
+
+function create() {
+    router.push(`${props.root}pages/create`);
```

Comment:
As I mentioned in https://github.com/galaxyproject/galaxy/pull/16209#issuecomment-1598327830 I think the issue is here:

```suggestion
    router.push("/pages/create");
```

The router should be smart enough to handle the prefixes so we don't have to use the `root` anymore. You can get rid of the props here too.